default.nix.mustache

```diff
@@ -1,36 +1,75 @@
 # This file was generated from `meta.yml`, please do not edit manually.
 # Follow the instructions on https://github.com/coq-community/templates to regenerate.
-
-{ pkgs ? (import <nixpkgs> {}), coq-version-or-url, shell ? false }:
-
+{
+  nixpkgs ? (if builtins.pathExists ./.nix/nixpkgs.nix then import ./.nix/nixpkgs.nix
+             # else fetchTarball https://github.com/NixOS/nixpkgs-channels/archive/502845c3e31ef3de0e424f3fcb09217df2ce6df6.tar.gz),
+             else null),
```

Comment:
Could it make sense to have a `else import <nixpkgs> {}` (until a better default is proposed) as in the previous version of this file?

Reply:
I'm not sure yet